openpower/package/openpower-pnor/openpower-pnor.mk

```diff
@@ -4,8 +4,14 @@
 #
 ################################################################################
 
-OPENPOWER_PNOR_VERSION ?= 1c47d2389691a9c343659c6a2ed794084be550ff
-OPENPOWER_PNOR_SITE ?= $(call github,ibm-op-release,pnor,$(OPENPOWER_PNOR_VERSION))
+# Will update _VERSION and _SITE and will remove _METHOD once PNOR pull request is finished
+#OPENPOWER_PNOR_VERSION ?= 1c47d2389691a9c343659c6a2ed794084be550ff
+#OPENPOWER_PNOR_SITE ?= $(call github,ibm-op-release,pnor,$(OPENPOWER_PNOR_VERSION))
+
+OPENPOWER_PNOR_VERSION = mab
+OPENPOWER_PNOR_SITE = /esw/san5/mbaiocch/OP_REPOS/W_op940/Mike_Pnor/mab_pnor
+OPENPOWER_PNOR_SITE_METHOD = local
+
```

Comment:
This needs to be removed/changed?

Reply:
Yes, I put a comment to this effect on line 7 above in that file.

Reply:
instead of messing with the makefile, prefer using OPENPOWER_PNOR_OVERRIDE_SRCDIR=/path/to/local/pnor in op-build/output/local.mk file.

That makes it cleaner for local testing